src/setu.js

```diff
@@ -50,6 +50,15 @@ class Setu {
         }
     };
 
+    setBillExpiry(platformBillID=null) {
+        if (this.checkIfValuesExist()) {
+            return requestHelper.post(
+                this.endpoints['bill-expiry']+platformBillID+'/expire',
```

Comment:
can we have something like

this.endpoints['bill-expiry'] + `/${platformBillID}/expire'

Reply:
done